MapboxMobileEventsTests/MMEEventsManagerTests.mm

```diff
@@ -831,14 +831,21 @@ @interface MMEEventsManager (Tests) <MMELocationManagerDelegate>
             
             it(@"enqueues the correct event", ^{
                 CLLocation *location = [[CLLocation alloc] initWithLatitude:visit.coordinate.latitude longitude:visit.coordinate.longitude];
-                NSDictionary *attributes = @{MMEEventKeyCreated: [eventsManager.dateWrapper formattedDateStringForDate:[location timestamp]],
+                NSDictionary *attributes = @{MMEEventKeyCreated: @"2018-05-30T21:53:08.720+0000",
```

Comment:
Have you consider using an `NSDate` + `NSDateFormatter` instead of hardcoding a string date?

Reply:
Right, that was what I was using before and what was causing an issue. When the enqueue event is created with the `didVisit:` method, then a "created" time is added to the dictionary via NSDate. After that the `expectedVisitEvent`'s "created" attribute was added at nearly the same time (via NSDate) but would sometimes be off by a second and cause the test to fail.

Reply:
I was thinking more into a 'static' date you could use on both cases:
```
NSDictionary *attributes = @{MMEEventKeyCreated: [eventsManager.dateWrapper formattedDateStringForDate:[location timestamp]],
```
And
```
[tempDict setObject:[eventsManager.dateWrapper formattedDateStringForDate:[location timestamp]] forKey:@"created"];
```

`[location timestamp]` would be the same in both cases right?